.github/workflows/phgv2_ci.yml

```diff
@@ -26,7 +27,7 @@ jobs:
           distribution: temurin
           java-version: 17
       - name: Build with Gradle
-        run: ./gradlew clean build -x koverVerify
```

Comment:
I thought that flag ran code coverage?

Reply:
The -x means exclude which the koverVerify is what requires the minimum coverage.  So we were skipping it previously.  Then running it in the next step to get the xml report.